### PR TITLE
Autocorrect (some) RuboCop TODOs

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

File: src/api/app/controllers/source_controller.rb [hunk 1]
```diff
@@ -192,13 +192,12 @@ def package_command
...
       # even when we can create the package, an existing instance must be checked if permissions are right
       @project = Project.get_by_name(@target_project_name)
-      # rubocop:disable Layout/LineLength
       if !PACKAGE_CREATING_COMMANDS.include?(@command) || Package.exists_by_project_and_name(@target_project_name,
                                                                                              @target_package_name,
                                                                                              follow_project_links: SOURCE_UNTOUCHED_COMMANDS.include?(@command))
         validate_target_for_package_command_exists!
       end
-      # rubocop:enable Layout/LineLength
+
```
Comment (DavidKang) [inline src/api/app/controllers/source_controller.rb:200]: This empty line is really needed?

File: src/api/app/helpers/maintenance_helper.rb [hunk 2]
```diff
@@ -237,9 +236,8 @@ def copy_single_binary(arch, target_repository, source_project_name, source_pack
     cp_params[:setupdateinfoid] = update_info_id if update_info_id
     cp_params[:setrelease] = setrelease if setrelease
     cp_params[:multibuild] = '1' unless source_package_name.include?(':')
-    # rubocop:disable Layout/LineLength
     cp_path = "/build/#{CGI.escape(target_repository.project.name)}/#{URI.escape(target_repository.name)}/#{URI.escape(arch.name)}/#{URI.escape(target_package_name)}"
-    # rubocop:enable Layout/LineLength
+
```
Comment (DavidKang) [inline src/api/app/helpers/maintenance_helper.rb:240]: This empty line is really needed?

File: src/api/app/controllers/source_controller.rb (hunk 1 above)
Comment (dmarcoux) [reply src/api/app/controllers/source_controller.rb:200]: No, I'll remove it :)

File: src/api/app/helpers/maintenance_helper.rb (hunk 2 above)
Comment (dmarcoux) [reply src/api/app/helpers/maintenance_helper.rb:240]: I'll leave it. I don't have a strong opinion on this one. It was corrected like this. Fine for you?

Comment (DavidKang) [reply src/api/app/helpers/maintenance_helper.rb:240]: Fine for me :slightly_smiling_face:

**Pull request merged**